go.sum

```diff
@@ -2749,6 +2749,7 @@ golang.org/x/sys v0.0.0-20210908233432-aa78b53d3365/go.mod h1:oPkhp1MJrh7nUepCBc
 golang.org/x/sys v0.0.0-20210917161153-d61c044b1678/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
 golang.org/x/sys v0.0.0-20210927094055-39ccf1dd6fa6/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
 golang.org/x/sys v0.0.0-20211019181941-9d821ace8654/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
+golang.org/x/sys v0.0.0-20211019181941-9d821ace8654/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
```

Comment:
These lines are duplicates.

Reply:
Removed duplicate line per request. Is there a `make` command handles tidying up go.mod/go.sum? I'm surprised `go mod tidy` allows for duplicates. It doesn't strike me as a file we should manually edit.

For example, the [current master branch go.sum lists github.com/bugsnag/bugsnag-go v0.0.0-20141110184014-b1d153021fcd/go.mod h1:2oa8nejYd4cQ/b0hMIopN0lCRxU0bueqREvZLWFrtK8=](https://github.com/influxdata/telegraf/blob/master/go.sum#L479-L482) 4 times

Reply:
Seems to be an open issue: https://github.com/golang/go/issues/28456

Reply:
Hi @Hipska,
We're so close.
Is there anything we can do to get this merged?
Thanks!